src/test/java/org/broadinstitute/hellbender/tools/walkers/CombineGVCFsIntegrationTest.java

```diff
@@ -424,4 +435,117 @@ public void testCombineGvcfsWithMnps() throws Exception {
         args.addVCF(getTestFile("mnp.g.vcf"));
         runCommandLine(args);
     }
+
+    @Test
+    public void testCombineSomaticGvcfs() throws Exception {
+        final File output = createTempFile("combinegvcfs", ".vcf");
+        final ArgumentsBuilder args = new ArgumentsBuilder();
+        args.addReference(new File(b37Reference));
+        args.addOutput(output);
+        args.addVCF(getTestFile("NA12878.MT.filtered.g.vcf"));
+        args.addVCF(getTestFile("NA19240.MT.filtered.g.vcf"));
+        args.addBooleanArgument(CombineGVCFs.USE_SOMATIC_LONG_NAME, true);
+        runCommandLine(args);
+
+        final List<VariantContext> actualVC = getVariantContexts(output);
+        for (final VariantContext vc : actualVC) {
+            if (vc.getAlternateAlleles().size() > 1) {  //if there's a real ALT
+                Assert.assertTrue(vc.filtersWereApplied());  //filtering should happen during combine
+            }
+            else {
+                Assert.assertFalse(vc.filtersWereApplied());
+            }
+
+            //MT:302 has an alphabet soup of alleles in the GVCF
+            if (vc.getStart() == 302) {
+                Assert.assertEquals(vc.getNAlleles(), 9);
+            }
+
+            //make sure phasing is retained
+            if (vc.getStart() == 317 || vc.getStart() == 320) {
+               VariantContextTestUtils.assertGenotypeIsPhasedWithAttributes(vc.getGenotype(1));
+            }
+
+            //MT:4769 in combined GVCF has uncalled alleles, but we should keep them around
+            if (vc.getStart() == 4769) {
+                Assert.assertEquals(vc.getNAlleles(), 4);
+                Assert.assertTrue(vc.getAlternateAlleles().contains(Allele.create("G", false)));
+                Assert.assertTrue(vc.getAlternateAlleles().contains(Allele.create("T", false)));
+                Assert.assertTrue(vc.getAlternateAlleles().contains(Allele.NON_REF_ALLELE));
+            }
+
+            //check genotype filtering
+            if (vc.getStart() == 14872) {
+                Assert.assertTrue(!vc.isFiltered());
+                Assert.assertTrue(!vc.getGenotype(0).isFiltered());
+                Assert.assertTrue(vc.getGenotype(1).getFilters().contains("t_lod"));
```

Comment:
`"t_lod"` should be a constant from `GATKVCFConstants.TUMOR_LOD_FILTER_NAME` (especially if we ever plan on changing it to lod).

Reply:
Good point.  It's actually about to change, in fact.